ql/src/java/org/apache/hadoop/hive/ql/optimizer/physical/Vectorizer.java

```diff
@@ -51,6 +51,7 @@
 import org.apache.hadoop.hive.ql.exec.vector.reducesink.*;
 import org.apache.hadoop.hive.ql.exec.vector.udf.VectorUDFArgDesc;
 import org.apache.hadoop.hive.ql.io.AcidUtils;
+import org.apache.hadoop.hive.ql.udf.*;
```

Comment:
wildcard imports

Reply:
Removed wildcard import and imported only necessary newly added UDF classes